src/databricks/labs/remorph/cli.py

```diff
@@ -33,12 +33,14 @@ def raise_validation_exception(msg: str) -> Exception:
 
 
 @remorph.command
+# pylint: disable=too-many-arguments
```

Comment:
Nit: May be chaging the `pyproject.toml` file max-args is better in this case than file level disabling.

Reply:
Done